pkg/virt-launcher/virtwrap/device/hostdevice/sriov/hostdev_test.go

```diff
@@ -164,6 +165,58 @@ var _ = Describe("SRIOV HostDevice", func() {
 			Expect(devices, err).To(Equal([]api.HostDevice{expectHostDevice1}))
 		})
 
+		DescribeTable("create two devices with custom guest PCI address",
+			func(iface1GuestPCIAddress, iface2GuestPCIAddress string) {
+				iface1 := newSRIOVInterface(netname1)
+				iface1.PciAddress = iface1GuestPCIAddress
```

Comment:
I would extract it to helper func (similar to `newSRIOVInterfaceWithBootOrder`)

Reply:
Done.